### DISCUSSION TITLE
getting an error while looping the data

### DISCUSSION THREAD

**Discussion by dinesh2216** [mrdbourke/pytorch-deep-learning#1021] [Q&A]
Hello i am watching na 25hr pytorch course

code
'' epochs = 1
for epoch in range(epochs):
  model_1.train()
  y_pred = model_1(X_train)
  loss = loss_fun(y_pred, y_train)
  optimizer.zero_grad()
  loss.backward()
  optimizer.step()
![Screenshot 2024-07-24 124111](https://github.com/user-attachments/assets/0bae26da-d103-45dc-ac07-c0665085f220)
''

**Comment by Acquiesense** [comment]
loss = loss_fn not loss_fun, unless you've changed the name of the variable before the for loop.
Idk about the Tensor error
